easybuild/easyconfigs/p/Python/Python-2.7.9-foss-2015a-bare.eb

```diff
@@ -17,8 +17,7 @@ dependencies = [
     ('zlib', '1.2.8'),
     ('libreadline', '6.3'),
     ('ncurses', '5.9'),
-#   ('OpenSSL', '1.0.1k'),  # OS dependency should be preferred if the os version is more recent then this version, it's
-#   nice to have an up to date openssl for security reasons
```

Comment:
keep this please, people should be able to easily switch to an EB-provided OpenSSL